kube-deploy/roles/kube-custom/tasks/main.yml

```diff
@@ -0,0 +1,51 @@
+ # Copyright 2016, JinkIT, and it's Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+- name: Update custom image for kube controller
+  lineinfile:
+    dest=/etc/kubernetes/manifests/kube-controller-manager.json
+    regexp="image"
+    backrefs=yes
+    line="\t"image\"{{ ":" }}{{" "}}"{{ controller_custom_image }}",\"
+    state=present
+  ignore_errors: True
+
+#- name: Update custom image for etcd
```

Comment:
ah, gotcha! that makes sense. seemed like something something was still in progress from what i could tell.